pkgs/tools/filesystems/catcli/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonApplication
+, docopt, anytree 
+}:
+
+buildPythonApplication rec {
+
+  pname = "catcli";
+  version = "v0.5.13";
+  
+  src = fetchFromGitHub {
+    owner = "deadc0de6";
+    repo = pname;
+    rev = version;
+    sha256 = "04mrkqmm2c8fhi1h1hddc4yh3dqhcvkmcwzhj0ggn34v7wavgb5i";
+  };
+
+  propagatedBuildInputs = [ docopt anytree ];
+
+  postPatch = '' patchShebangs . '';
```

Comment:
what necessitated this?
it seems to work without it (though i am not familiar with this tool)

Reply:
[here's](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/patch-shebangs.sh#L14) where I was pointed to when I asked. Apparently it's recommended to catch subtle path-related bugs (which could come up later if new deps are introduced). I don't think it's doing much now but it's also not harming anything.

Reply:
the `fixupPhase` already does this
going by the fact that without this line the wrapped script has a shebang to python in the nix store, i'm guessing it also runs for `buildPythonApplication`